authors/apps/ratings/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from . import views
+
+app_name = "articles"
```

Comment:
I think that the `app_name` here should be ratings since we are currently in the ratings app.